python/tstUtilities.py

```diff
@@ -417,6 +417,29 @@ def processRequirementLines(api, pieces, triggerCharacter):
 
     return returnData
 
+def processSubprogramLines(api, pieces, triggerCharacter, additionalParams):
```

Comment:
Ok so if you are only using this to pass the unit name, then change all of the parameter names etc to unitName, not additionalParams.   Additionally additionalParams is a bad name because there is only one right?

Reply:
At the moment, we would only need the `<unit>`, but I thought to make it as an object so that for future implementations I can simply add new params to this JSON. 

I changed it now to be only `unit`. In case we will need more, I could enhance it.